### PR TITLE
Sean / Landing page for DP2P

### PR DESCRIPTION
**Pull request by sean-binary**
# Description
New landing pages for DP2P

Summary

Changes:

-   Added 2 new versions of landing page for DP2P

## Type of change

-   [ ] Bug fix
-   [X] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [x] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/p2p/_availability.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+const Section = styled(SectionContainer)`
+    display: flex;
+    flex-direction: column;
+    padding: 8rem 12rem;
+    max-width: 1600px;
+    margin: 0 auto;
+    align-items: center;
+    justify-content: center;
+    box-shadow: inset 0 1px 0 0 #f2f3f4;
```
Comment (mustofa-binary) [inline src/pages/p2p/_availability.js:25]: you can use --color-grey-8 css variables for this

File: src/pages/p2p/_availability.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    background-color: var(--color-white);
+    width: 48.6rem;
+    position: relative;
+
+    @media ${device.laptopM} {
+        min-height: 22rem;
+        height: 100%;
+    }
+    @media ${device.tablet} {
+        margin: 0 0 24px 0;
```
Comment (mustofa-binary) [inline src/pages/p2p/_availability.js:42]: margin bottom?

File: src/pages/p2p/_availability.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        max-height: unset;
+        padding: 24px 24px 24px;
+        max-width: 328px;
+
+        :first-child {
+            margin: 0 0 24px 0;
```
Comment (mustofa-binary) [inline src/pages/p2p/_availability.js:48]: same here

File: src/pages/p2p/_exchange-steps.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+    width: 100%;
+`
+
+const Section = styled(SectionContainer)`
+    display: flex;
+    flex-direction: column;
+    padding: 8rem auto;
+    align-items: center;
+    justify-content: center;
+    box-shadow: inset 0 1px 0 0 #f2f3f4;
```
Comment (mustofa-binary) [inline src/pages/p2p/_exchange-steps.js:41]: can use --color-grey-8 too here

File: src/pages/p2p/_exchange-steps.js [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+const CardContainer = styled(Flex)`
+    @media ${device.laptopM} {
+        max-width: 58rem;
+        flex-wrap: wrap;
+    }
+`
+const Card = styled(Flex)`
+    flex-direction: column;
+    min-width: 38.4rem;
+    max-width: 38.4rem;
```
Comment (mustofa-binary) [inline src/pages/p2p/_exchange-steps.js:56]: width?

File: src/pages/p2p/_exchange-steps.js [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+    @media ${device.mobileL} {
+        font-size: 20px;
+        margin-bottom: 8px;
+        margin-top: 8px;
+        line-height: 1.2;
+    }
+`
+const StyledText = styled(Text)`
+    font-size: 24px;
```
Comment (mustofa-binary) [inline src/pages/p2p/_exchange-steps.js:95]: 2.4rem

File: src/pages/p2p/_exchange-steps.js [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+            <StyledHeader align="center" mb="4rem" as="h2">
+                {localize('Exchange in 3 easy steps')}
+            </StyledHeader>
+            <CardContainer>
+                <Card>
+                    <Row>
+                        <RightAd />
+                    </Row>
+                    <Row>
+                        <Column>
```
Comment (mustofa-binary) [inline src/pages/p2p/_exchange-steps.js:113]: you can use <ol> <li> here instead

File: src/pages/p2p/_availability.js (hunk 1 above)
Comment (sean-binary) [reply src/pages/p2p/_availability.js:25]: Alright, will be incorporated

File: src/pages/p2p/_availability.js (hunk 2 above)
Comment (sean-binary) [reply src/pages/p2p/_availability.js:42]: Alright, will be incorporated

File: src/pages/p2p/_availability.js (hunk 3 above)
Comment (sean-binary) [reply src/pages/p2p/_availability.js:48]: Alright, will be incorporated

File: src/pages/p2p/_exchange-steps.js (hunk 4 above)
Comment (sean-binary) [reply src/pages/p2p/_exchange-steps.js:41]: Alright, will be incorporated

File: src/pages/p2p/_exchange-steps.js (hunk 6 above)
Comment (sean-binary) [reply src/pages/p2p/_exchange-steps.js:95]: Alright, will be incorporated

File: src/pages/p2p/_exchange-steps.js (hunk 5 above)
Comment (sean-binary) [reply src/pages/p2p/_exchange-steps.js:56]: Will replace them with widths

File: src/pages/p2p/index.js [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+        image_name: 'web_and_mobile',
+        image_alt: localize('Trade Types'),
+    },
+]
+
+const P2P = () => {
+    const [is_mobile, setMobile] = useState('')
+
+    const handleResizeWindow = () => {
```
Comment (mustofa-binary) [inline src/pages/p2p/index.js:43]: we can use show component here instead?

Comment (sean-binary) [reply src/pages/p2p/index.js:43]: Alright, will be incorporated

File: src/pages/p2p/_hero.js [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+            <Wrapper>
+                <InformationWrapper height="unset" direction="column">
+                    <StyledHeader as="h1" weight={500}>
+                        {title}
+                    </StyledHeader>
+                    <HeroContent>
+                        <Header as="h1">{content}</Header>
+                    </HeroContent>
+                    <TryButton secondary="true" external is_deriv_app_link to="/cashier/P2P">
+                        {localize('Try P2P now')}
```
Comment (kevinw-binary) [inline src/pages/p2p/_hero.js:182]: shouldn't it be `DP2P` instead of `P2P` ?

File: src/pages/p2p/_hero.js [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+        margin-bottom: 2rem;
+    }
+    @media ${device.mobileM} {
+        margin-bottom: 4rem;
+    }
+`
+
+const query = graphql`
+    query {
+        p2p_hero_background: file(relativePath: { eq: "p2p_hero_background.png" }) {
```
Comment (kevinw-binary) [inline src/pages/p2p/_hero.js:159]: If it's `dp2p` isnt it better to rename p2p to dp2p ?

File: src/components/layout/layout.js [hunk 11]
```diff
@@ -128,6 +129,10 @@ const Layout = ({ children, type, interim_type, padding_top, no_login_signup })
             Navigation = <NavCareers />
             FooterNav = <Footer no_language={true} />
             break
+        case 'p2p':
+            Navigation = <NavP2P />
+            FooterNav = <Copyright />
```
Comment (kevinw-binary) [inline src/components/layout/layout.js:134]: I think the copyright should have a white color fill or maybe try using the `copyrigt.svg`  and change the fill color using styled component

File: src/pages/p2p/_hero.js (hunk 9 above)
Comment (sean-binary) [reply src/pages/p2p/_hero.js:182]: Oh yes, you are right. Will have it changed.

File: src/pages/p2p/_hero.js (hunk 10 above)
Comment (sean-binary) [reply src/pages/p2p/_hero.js:159]: Got it, will have it changed.

File: src/components/layout/layout.js [hunk 12]
```diff
@@ -137,6 +152,8 @@ const Layout = ({ children, type, interim_type, padding_top, no_login_signup })
 Layout.propTypes = {
     children: PropTypes.node.isRequired,
     interim_type: PropTypes.string,
+    is_p2p_v1: PropTypes.bool,
```
Comment (mustofa-binary) [inline src/components/layout/layout.js:155]: i think this is too specific, can we name it to navigation_type instead?

Comment (sean-binary) [reply src/components/layout/layout.js]: Alright, will refactor it to a more general usage.

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**